public/docs/ts/latest/guide/component-styles.jade

```diff
@@ -294,54 +304,61 @@ code-example(format='').
   Because these files are co-located with the component,
   it would be nice to refer to them by name without also having to specify a path back to the root of the application.
-  
-  We'd *prefer* to write this:
-+makeExample('component-styles/ts/app/quest-summary.component.ts', 'urls')(format='.')
-:marked
-  We can't do that by default. Angular can't find the files and throws an error:
 
-  `EXCEPTION: Failed to load quest-summary.component.html`
+block module-id
+  :marked
+    We'd *prefer* to write this:
 
-  Why can't Angular calculate the HTML and CSS URLs from the component file's location? 
-  
-  Unfortunately, that location is not readily known. 
-  Angular apps can be loaded in many ways: from individual files, from SystemJS packages, or
-  from CommonJS packages, to name a few. 
-  With this diversity of load strategies, it's not easy to tell at runtime where these files actually reside.
-  
-  The only location Angular can be sure of is the URL of the `index.html` home page.
-  So by default it resolves template and style paths relative to the URL of `index.html`.
-  That's why we previously wrote our CSS file URLs with an `app/` base path prefix.
-  
-  Although this works with any code loading scheme, it is very inconvenient.
-  We move file folders around all the time during the evolution of our applications. 
-  It's no fun patching the style and template URLs when we do.
-  
-  ### *moduleId*
-  
-  We can change the way Angular calculates the full URL be setting the component metadata's `moduleId` property.
-  
-  If we knew the component file's base path, we'd set `moduleId` to that and 
-  let Angular construct the full URL from this base path plus the CSS and template file names.
-  
-  Our challenge is to calculate the base path with minimal effort. 
-  If it's too hard, we shouldn't bother; we should just write the full path to the root and move on.
+  +makeExample('component-styles/ts/app/quest-summary.component.ts', 'urls')(format='.')
 
-  Fortunately, *certain* module loaders make it easy.
-  SystemJS (starting in v.0.19.19) sets a special `__moduleName` variable to the URL of the component file.
-.alert.is-important
   :marked
-    Caution: we currently regard the *__moduleName* feature as experimental.
-:marked
-  Now it's trivial to set the `moduleId` to the `__moduleName` and write module-relative paths for style and template URLs.
-  
-+makeExample('component-styles/ts/app/quest-summary.component.ts','', 'app/quest-summary.component.ts')
+    We can't do that by default. Angular can't find the files and throws an error:
+
+    `EXCEPTION: Failed to load quest-summary.component.html`
+
+    Why can't Angular calculate the HTML and CSS URLs from the component file's location? 
+
+    Unfortunately, that location is not readily known. 
+    Angular apps can be loaded in many ways: from individual files, from SystemJS packages, or
```

Comment:
It isn't This is the TS prose. It is difficult to see but this text is under the block `module-id` (see [Line 313](https://github.com/angular/angular.io/pull/1320/files#diff-452f86bc62408dcd7a93a8ce1ad3dacbR313)).
The corresponding Dart prose is [here](https://github.com/angular/angular.io/pull/1320/files#diff-1291d1db3d442dba259a6c4d2d3feea6R30).